plugins/modules/system/filesystem.py

```diff
@@ -144,6 +159,22 @@ def create(self, opts, dev):
             cmd = "%s %s %s '%s'" % (mkfs, self.MKFS_FORCE_FLAGS, opts, dev)
         self.module.run_command(cmd, check_rc=True)
 
+    def wipefs(self, dev):
+        if self.module.check_mode:
+            return
+
+        try:
+            # wipefs comes with util-linux package
+            wipefs = self.module.get_bin_path('wipefs', required=True)
+            cmd = [wipefs, "--all", "%s" % dev]
```

Comment:
Why not:
```suggestion
            cmd = [wipefs, "--all", dev]
```
?

Reply:
because `dev = Device(module, dev)` in main() makes that `dev` in the list is not a supported type (string or byteObject), since it is `Device`; the format used in create() works, and moving to an array without forcing type string (as in create()) raises an exception.

Reply:
Ah, I missed that one. Thanks for pointing it out!